src/phases/mystery-encounter-phases/battle-start-cleanup-phase.ts

```diff
@@ -62,7 +61,7 @@ export class MysteryEncounterBattleStartCleanupPhase extends Phase {
     /** The total number of legal player Pokemon that aren't currently on the field */
     const legalPlayerPartyPokemon = legalPlayerPokemon.filter((p) => !p.isActive(true));
     if (!legalPlayerPokemon.length) {
-      globalScene.unshiftPhase(new GameOverPhase());
+      globalScene.gameOver({ clearPhaseQueue: true });
```

Comment:
```suggestion
      globalScene.gameOver({ clearPhaseQueue: false });
```
also here, clearPhaseQuere wasn't being called before?